README.md

```diff
@@ -1,9 +1,69 @@
 # Polymer Linter
 
-Catch errors in your Polymer project before even running your code.
+Analyze your Polymer project for errors.
```

Comment:
Feels a little vague. Looking around at other descriptions, how about something like:

"A library for finding and reporting on suspicious patterns and errors in web projects."